src/components/Tag/core/tag.module.scss

```diff
@@ -0,0 +1,52 @@
+@use 'sass:map';
+@use '../../../style/variables/colors.scss';
+@use '../../../style/variables/spaces.scss';
+@use '../../../style/mixins/font-config.scss';
+
+.tag {
+  color: map.get(colors.$colors, 'base-white');
+  border-radius: map.get(spaces.$spaces, '1');
+
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+
+  display: flex;
+  flex-direction: row;
+  align-items: center;
```

Comment:
These are useless since you already used the helpers `f`, `f-direction-row` & `f-ai-center`